### PR TITLE
Feature Web Version

### PR DESCRIPTION
**Pull request by joshuaoivie**
Dear Code Reviewer,

This pull request contains the following changes

-Refactor HTML structure for Web
-Add Web Style to Hero, Portfolio, About Me and Contact Section
-Add style for decorative elements
-Fixed lint errors

Warm Regards,
Joshua Ivie

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,10 +26,32 @@
...
+      <img src="./assets/img/hero/Ellipse-4.svg" class="shape eclipse-4" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-5.svg" class="shape eclipse-5" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-6.svg" class="shape eclipse-6" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-7.svg" class="shape eclipse-7" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-8.svg" class="shape eclipse-8" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-11.svg" class="shape eclipse-11" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-12.svg" class="shape eclipse-12" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-13.svg" class="shape eclipse-13" alt="shape" />
+      <img src="./assets/img/hero/Ellipse-14.svg" class="shape eclipse-14" alt="shape" />
+    </div>
```
Comment (Whoistolu) [inline index.html]: - [x] [REQUIRED] Please note that all these images are not supposed to be added to your HTML with `img` tags, they are supposed to be added into your CSS as background images. Kindly add them. Also, note that some of them are not displayed on your first page.

File: index.html [hunk 2]
```diff
@@ -47,130 +69,160 @@ <h1 class="title">Hello There! 👋 <br /> Welcome to my Website!</h1>
...
+    <div class="footer-shapes">
+      <img src="./assets/img/footer/Rectangle-4.svg" class="shape rectangle-4" alt="shape" />
+      <img src="./assets/img/footer/Rectangle-6.svg" class="shape rectangle-6" alt="shape" />
+      <img src="./assets/img/footer/Rectangle-7.svg" class="shape rectangle-7" alt="shape" />
+      <img src="./assets/img/footer/Ellipse-3.svg" class="shape ellipse-3" alt="shape" />
+      <img src="./assets/img/footer/Ellipse-5.svg" class="shape ellipse-5" alt="shape" />
+      <img src="./assets/img/footer/Ellipse-11.svg" class="shape ellipse-11" alt="shape" />
+      <img src="./assets/img/footer/Ellipse-12.svg" class="shape ellipse-12" alt="shape" />
+      <img src="./assets/img/footer/Ellipse-13.svg" class="shape ellipse-13" alt="shape" />
+    </div>
```
Comment (Whoistolu) [inline index.html]: - [x] [REQUIRED] Please refer to my previous comment. Add these SVGs as background images and use background properties to align them (background-position, background-repeat etc)

File: index.html [hunk 3]
```diff
@@ -200,6 +252,23 @@ <h3 class="title">Skills</h3>
       </form>
```
Comment (Whoistolu) [inline index.html]: - [x] [REQUIRED] Kindly fix your form. the input field width needs to be reduced. Please refer to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
Also, reduce the size of the c-shaped SVG icon as it is currently altering the text.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

File: assets/css/style.css [hunk 4]
```diff
@@ -524,3 +694,208 @@ svg {
...
+
+  .footer .footer-content {
+    max-width: 600px;
+    margin: auto;
+  }
+
+  .footer .contact-form {
+    padding: 0 24px;
+  }
+}
```
Comment (Whoistolu) [inline assets/css/style.css]: - [x] [REQUIRED]You have done a great job concerning the desktop version, however, you need to stick to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible.
The landing page looks different,
The card/work sections need to be centered. You can use margin-left and right: auto or give it a specific margin on the left and right.
Your contact section also looks different from the one provided in the template.
Kindly ensure that your entire design looks more like the figma design provided.

Take a look at these images:

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 | 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 | 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
 |

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @joshuaivie ,

You have done a good job so far! 👍🏽  However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A
Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Deepakdanger — APPROVED** [review]
Hi <STUDENT_MENTION>, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] You have used the different div for adjusting the background picture, which is a bad coding technique, you should use the background positioning, and sizing in CSS.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**